searchlib/src/vespa/searchlib/attribute/postinglistsearchcontext.hpp

```diff
@@ -95,6 +99,7 @@ struct PostingListSearchContextT<DataT>::FillPart : public vespalib::Runnable {
             if (__builtin_expect(key < _docIdLimit, true)) { _bv->setBit(key); }
         });
     }
+    const vespalib::Doom       _doom;
```

Comment:
Was it intentional to take a copy here?

Reply:
Yes, you avoid an indirection as it wraps a reference.